### PR TITLE
Job Index View

### PR DESCRIPTION
**Pull request by Kealii**
User Can: 
See jobs ordered by date
See tags for technologies related to the job
See company name related to the job
Click through a numbered list of pages
See which page they're currently on
Go to the first page
Go to the last page
Go to the next page
Go to the previous page
All components are tested. If you wouldn't mind taking a look @rrgayhart , I think it's ready to be merged in. I think the styling is also a big improvement! Simple, but not claw my eyes out simple.

### CODE REVIEW

**Comment by Kealii** [comment]
closes #13

**Comment by Kealii** [comment]
Preview

**Comment by amcrawford** [comment]
This look beautiful!

I took a look at the files changes; we're passing a lot of variables into the index view through the Home Controller. Could we possibly consolidate @jobs and @jobs_count and do that calculation in the view?

**Comment by Kealii** [comment]
Thanks for taking a look through the file changes! I definitely agree about the instance variables and I'll quash that. That was actually what my question yesterday was about and even though Meeka said it was fine, I was still hesitant to modify tests that I didn't write. But that's silly and Meeka said so and I'll fix it!

**Comment by rjohnson4444** [comment]
That looks good Mike! Yea it matches up nicely to the API. As far as I know it looks ready to merge.

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@
 !/log/.keep
 /tmp
 .byebug_history
+.idea
```
Comment (rrgayhart) [inline .gitignore]: Wat dis?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import 'bootstrap-sprockets';
```
Comment (rrgayhart) [inline app/assets/stylesheets/application.scss]: Yay for sass!

File: app/models/job.rb [hunk 3]
```diff
@@ -13,4 +13,12 @@ def assign_tech
     tech_matches = Technology.where(name: raw_technologies)
     self.technologies = tech_matches
   end
+
+  def company_name
```
Comment (rrgayhart) [inline app/models/job.rb]: These methods are being used for view logic - we might want to consider using [decorators](http://johnotander.com/rails/2014/03/07/decorators-on-rails/) or [View Objects](http://www.jetthoughts.com/blog/tech/2014/08/14/cleaning-up-your-rails-views-with-view-objects.html)

File: spec/controllers/home_controller_spec.rb [hunk 4]
```diff
@@ -14,7 +14,7 @@
       expect(assigns(:company_count)).to eq(3)
     end
 
-    it "populates an array of technology names" do
```
Comment (rrgayhart) [inline spec/controllers/home_controller_spec.rb]: Nice!

**Comment by rrgayhart** [comment]
This looks great and also, great work!

I think pulling the view helper logic out here would be important. I think you have two courses of action:
- If Ryan's work needs some of the things you set up here, I would fix the mergies and merge this then create an issue and start working on using decorators or view objects.
- If not, do the refactoring in this PR

File: .gitignore (hunk 1 above)
Comment (Kealii) [inline .gitignore]: This a Rubymine internal version control thing. It helps with telling other RM clients what the structure of the 'project' is. [Their docs are better at explaining](https://www.jetbrains.com/ruby/help/project.html)

Comment (rrgayhart) [inline .gitignore]: Nice!

**Comment by rrgayhart** [comment]
BTW the Draper Gem is a Jeff Casimir original - so it would make for a good blog post and you could probably ping him with questions

File: app/models/job.rb (hunk 3 above)
Comment (Kealii) [inline app/models/job.rb]: Thanks for these links! I really like the way decorators handle presentation so I'll work on that next :+1:

**Pull request closed without merging**